settings.gradle

```diff
@@ -0,0 +1,5 @@
+rootProject.name = 'taf'
+
+include 'api'
+include 'ui'
+include 'core'
```

Comment:
looks like we discussed that core module will be a separate git sub repo

Reply:
updated